resources/cluster-essentials/templates/psp-privileged.yaml

```diff
@@ -0,0 +1,24 @@
+---
+apiVersion: policy/v1beta1
+kind: PodSecurityPolicy
+metadata:
+  name: kyma.privileged
+  annotations:
+    helm.sh/hook: "pre-upgrade, pre-install"
+    helm.sh/hook-weight: "0"
+    helm.sh/hook-delete-policy: "before-hook-creation"
+spec:
+  allowPrivilegeEscalation: true
```

Comment:
I'm also not sure why do we need to have a privileged psp. I know Gardener does this, but I don't see its usage in the case of Kyma right now.

Reply:
I added it as it may be prefered/wanted by some OpenSource users, or some use cases (evaluation profile?). This approach simplifies this.

Reply:
Ok makes sense!